src/sagas/projects.js

```diff
@@ -87,6 +90,27 @@ export function* updateProjectSource() {
   yield* saveCurrentProject();
 }
 
+export function* loadAndBeautifyProjectSource() {
+  const state = yield select();
+  const currentProject = getCurrentProject(state);
```

Comment:
You can collapse these two lines into one by doing `const currentProject = yield select(getCurrentProject)`